bedwars-plugin/src/main/java/com/tomkeuper/bedwars/support/vault/NoChat.java

```diff
@@ -20,9 +20,10 @@
 
 package com.tomkeuper.bedwars.support.vault;
 
+import com.tomkeuper.bedwars.api.chat.IChat;
 import org.bukkit.entity.Player;
 
-public class NoChat implements Chat {
```

Comment:
I dont see that you removed the old Chat and Economy interfaces, is this correct?

Reply:
I have not yet eliminated the old Chat and Economy classes but I can do it

Reply:
They wont be used any more so clean up is good